ghost/core/core/server/services/auth/session/session-service.js

```
@@ -128,7 +128,8 @@ module.exports = function createSessionService({
         session.user_agent = req.get('user-agent');
         session.ip = req.ip;
 
-        if (!labs.isSet('staff2fa')) {
+        console.log('isStaffDeviceVerificationDisabled', isStaffDeviceVerificationDisabled());
```

ESLint

[error] Unexpected console statement.

(no-console)